ppr-ui/tests/unit/MhrInformation.spec.ts

```diff
@@ -394,4 +394,43 @@ describe('Mhr Information', () => {
     expect(accountInfoText).toContain(mockedAccountInfo.mailingAddress.street)
     expect(accountInfoText).toContain(mockedAccountInfo.mailingAddress.postalCode)
   })
+
+  it('should render TransferDetailsReview on Review screen', async () => {
+    setupCurrentHomeOwners()
+    wrapper.vm.$data.dataLoaded = true
+    await Vue.nextTick()
+
+    // set some test values for transfer details fields
+    const mhrTransferDetailsComponent = wrapper.findComponent(MhrInformation).findComponent(TransferDetails)
+    mhrTransferDetailsComponent.find(getTestId('declared-value')).setValue(123456)
+    mhrTransferDetailsComponent.find(getTestId('declared-value')).trigger('blur')
+    await Vue.nextTick()
+    mhrTransferDetailsComponent.find(getTestId('lease-own-checkbox')).setChecked()
+    await Vue.nextTick()
+
+    expect(wrapper.findComponent(TransferDetailsReview).exists()).toBeFalsy()
+
+    // go to Review screen
+    wrapper.find('#btn-stacked-submit').trigger('click')
+    await Vue.nextTick()
+    await Vue.nextTick()
+
+    // renders TranseferDetailsReview
+    expect(wrapper.findComponent(TransferDetailsReview).exists()).toBeTruthy()
+    const mhrTransferDetailsReviewComponent = wrapper.findComponent(TransferDetailsReview)
+
+    // displaying correct declared value
+    expect(mhrTransferDetailsReviewComponent.find('#declared-value-display')).toBeTruthy()
```

Comment:
Should have expected value, ie: `.exists()` of `.visible()`.

Reply:
good catch